internal/operator/operator_group.go

```diff
@@ -14,7 +14,7 @@ type OperatorGroupData struct {
 }
 
 func (o *operatorClient) CreateOperatorGroup(ctx context.Context, data OperatorGroupData, namespace string) (*operatorv1.OperatorGroup, error) {
-	logger.Debugf("Creating OperatorGroup %s in namespace %s", data.Name, namespace)
+	logger.Debug("creating OperatorGroup", "operatorGroup", data.Name, "namespace", namespace)
```

Comment:
Debug should be Debugw
operatorGroup should be operatorgroup (currently the standard is all lowercase for key names); unless we want to change this which would require other changes.